DifferentiationInterface/ext/DifferentiationInterfaceTapirExt/twoarg.jl

```diff
@@ -3,7 +3,9 @@ struct TapirTwoArgPullbackExtras{R} <: PullbackExtras
 end
 
 function DI.prepare_pullback(f!, y, ::AutoTapir, x, dy)
-    return TapirTwoArgPullbackExtras(build_rrule(f!, y, x))
+    rrule = build_rrule(f!, y, x)
+    # how to warm up here?
```

Comment:
Can you just call the method of `DI.value_and_pulback` defined a couple of lines below here?

Reply:
oh right, I was afraid it would create an infinite recursive loop but it won't